test/rubygems/helper.rb

```diff
@@ -299,7 +297,7 @@ def assert_contains_make_command(target, output, msg = nil)
     assert scan_make_command_lines(output).any? {|line|
       make = parse_make_command_line(line)
 
-      if make[:targets].include?(target)
+      if target == '' || make[:arguments].include?(target)
```

Comment:
Sorry. The returning true always if the given make target is empty is not appropriate. I need to think a better logic.

Reply:
But the problem is that I have no idea about how to parse only the make targets skipping some patterns of the make options (`-jN`, `-i`, `-j N` or `--jobs=N`). Perhaps, we can just skip the tests when we find the "-" in the make command line, supposing it as the make options?